Source/driver/Castro.cpp

```diff
@@ -598,9 +598,9 @@ Castro::Castro (Amr&            papa,
 
     buildMetrics();
 
-    // initialize the C++ values of the runtime parameters
     if (do_init_probparams == 0) {
```

Comment:
Well now this variable name needs to be updated.

Reply:
done